packages/block-editor/src/components/use-moving-animation/index.js

```diff
@@ -66,8 +67,15 @@ function useMovingAnimation(
 		clientTop: 0,
 	} );
 
-	const previous = ref.current ? getAbsolutePosition( ref.current ) : null;
 	const scrollContainer = useRef();
+	const previous = useMemo(
+		() => ( ref.current ? getAbsolutePosition( ref.current ) : null ),
```

Comment:
Wrapped this in `useMemo` as well. No need to fetch on every render.

Reply:
I'm pretty sure this will produce bugs. say you type on a previous block more than a single line of text and you reorder afterward, it will use the original position and not the updated one (after typing)

Reply:
Oh I guess `triggerAnimationOnChange` ensure it's triggered  before the animation, so  it's fine;

Reply:
Yes, it will always be recalculated before the effect is run.